pynance/textimporter_test.py

```diff
@@ -244,6 +250,147 @@ def test_csv_importer_read_dkbvisa_text(self):
                     "REWE Markt GmbH ZW"]
         self.assertListEqual(expected, list(result_df["text"].values))
 
+    def test_dkbvisa_StringIO_import(self):
+        csv_desc = SupportedCsvTypes.DKBVisa
+
+        content = u"""
+            "Kreditkarte:";"3546********6546";
+
+            "Zeitraum:";"letzten 60 Tage";
+            "Saldo:";"465,33 EUR";
+            "Datum:";"28.01.2019";
+            """\
```

Comment:
Is this concatenation of multi-line and single line strings necessary? Couldn't it be a single multi-line string?

Reply:
was necessary to abide by line length laws 😄